packages/cli/src/commands/list.ts

```diff
@@ -271,18 +292,32 @@ export default async function main(client: Client) {
   client.output.print(
     `${table(
       [
-        ['project', 'latest deployment', 'state', 'age', 'username'].map(
-          header => chalk.dim(header)
-        ),
+        (showUsername
+          ? [
+              'Age',
+              inspect ? 'Inspect URL' : 'Deployment URL',
+              'Status',
+              'Duration',
+              'Username',
+            ]
+          : [
+              'Age',
+              inspect ? 'Inspect URL' : 'Deployment URL',
+              'Status',
+              'Duration',
+            ]
```

Comment:
You could simplify this logic by putting these into a variable above like so:

```js
const headers = ['Age', inspect ? 'Inspect URL' : 'Deployment URL', 'Status', 'Duration'];
if (showUsername) headers.push('Username');
```

Reply:
Thanks!